backend/src/utils/ffmpeg_utils.py

```diff
@@ -454,21 +461,40 @@ def find_filters(video_path: str) -> List[Filters]:
     return global_filters
 
 
-def find_transi(video_path: str) -> Transition:
+def find_transi(video_path: str) -> Tuple[Transition, Transition]:
     """
-    Find the transition for the video and return its name and the time it last
+    Find the starting and ending transition for the video
     """
-    TRANSI = get_transitions()
-    global_transi: Tuple[str, int] = ("", 0)
-    if "transi" in TRANSI:
-        global_transi = TRANSI["transi"]
+    TRANSI = get_transition()
+    in_transi: Tuple[str, Union[None, Transition]] = ("", None)
+    out_transi: Tuple[str, Union[None, Transition]] = ("", None)
+
+    # Get global starting and ending transition
+    if "transitions" in TRANSI:
+        if "in" in TRANSI["transitions"] and TRANSI["transitions"]["in"] != {}:
+            key = next(iter(TRANSI["transitions"]["in"]))
+            in_transi = key, TRANSI["transitions"]["in"][key]
+        if "out" in TRANSI[
+                "transitions"] and TRANSI["transitions"]["out"] != {}:
+            key = next(iter(TRANSI["transitions"]["out"]))
+            out_transi = key, TRANSI["transitions"]["out"][key]
+
     video_name = os.path.split(video_path)[-1]
     no_ext = io.remove_extension(video_name)
+
+    # Get specific starting and ending transition for the current video (if they exist)
     if "clips" in TRANSI:
         if no_ext in TRANSI["clips"]:
-            global_transi = TRANSI["clips"][no_ext]
-
-    return Transition(global_transi[0], global_transi[1])
+            if "in" in TRANSI["clips"][
+                    no_ext] and TRANSI["clips"][no_ext]["in"] != {}:
+                key = next(iter(TRANSI["clips"][no_ext]["in"]))
+                in_transi = key, TRANSI["clips"][no_ext]["in"][key]
+            if "out" in TRANSI["clips"][
+                    no_ext] and TRANSI["clips"][no_ext]["out"] != {}:
+                key = next(iter(TRANSI["clips"][no_ext]["out"]))
+                out_transi = key, TRANSI["clips"][no_ext]["out"][key]
+
+    return Transition(*in_transi), Transition(*out_transi)
```

Comment:
Once we have name tupple, pretty sure you can do that

```suggestion
    return Transition(**in_transi), Transition(**out_transi)
```